src/constants/label.js

```diff
@@ -217,7 +338,7 @@ export const localizedNameWithLocalGloss = [
       ["var", "localizedCollator"],
     ],
     // ...just pick one.
-    ["var", "localizedName"],
+    ["format", listValuesExpression(["var", "localizedName"], "\n")],
```

Comment:
The return value of this method is so complex that the style-spec package (used by the unit tests) is apparently unable to upcast it to a `format` expression, leading to an error when it sees the direct `format` expression in the else slot below. Interestingly, GL JS has no such problem at runtime. This reassures the expression parser that each branch of the `case` expression will resolve to a `format` expression.